### PR TITLE
set default value for images-root

### PR DESCRIPTION
**Pull request by jpm38**
## Checklist before requesting a review
- [ ] Document introduced changes / variables in role README.md if any
- [x ] Increment role version in vars/main.yml (a.b.c: +1 to b for a feature added, +1 to c for a bug fix)
- [x ] Update changelog inside role README.md
- [ ] If not present, please also add your name in the related collection authors list in galaxy.yml

### CODE REVIEW

File: collections/infrastructure/roles/pxe_stack/templates/Ubuntu/equipment_profile_preseed.ipxe.j2 [hunk 1]
```diff
@@ -25,13 +25,14 @@ set node-main-network-interface auto
...
+set images-root http://${next-server}/{{ pxe_stack_images_root }}
+set images-root-iso ${images-root}/{{ pxe_stack_images_root_iso }}
 set repository-root http://${next-server}/repositories/${eq-repositories-environment}/${eq-distribution}/${eq-distribution-version}/${eq-architecture}/
-set preseed-path http://${next-server}/pxe/equipment_profiles//${eq-equipment-profile}.preseed.cfg
+set kickstart-path http://${next-server}/pxe/equipment_profiles/${eq-equipment-profile}.kickstart.cfg
 
 echo | images-root: ${images-root}
 echo | repository-root: ${repository-root}
-echo | preseed-path: ${preseed-path}
+echo | kickstart-path: ${kickstart-path}
```
Comment (oxedions) [inline collections/infrastructure/roles/pxe_stack/templates/Ubuntu/equipment_profile_preseed.ipxe.j2:35]: Can you restore this to preseed ?
Its ubuntu file, not kickstart

**Review by oxedions — CHANGES_REQUESTED** [review]
Thanks a lot for this PR. Apologies for the delay of my review.

Can you just fix the kickstrat->preseed as requested ? :blush:

File: collections/infrastructure/roles/pxe_stack/templates/Ubuntu/equipment_profile_preseed.ipxe.j2 (hunk 1 above)
Comment (jpm38) [reply collections/infrastructure/roles/pxe_stack/templates/Ubuntu/equipment_profile_preseed.ipxe.j2:35]: I will do ! thanks

**Comment by jpm38** [comment]
Close this PR as the comment has been integrated and merged in #963 PR by oxedions.

**Pull request closed without merging**